packages/dart/build.sh

```diff
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="Dart is a general-purpose programming language"
 TERMUX_PKG_LICENSE="BSD"
 TERMUX_PKG_LICENSE_FILE="sdk/LICENSE"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.3.1
+TERMUX_PKG_VERSION=3.3.2
 TERMUX_PKG_AUTO_UPDATE=false
```

Comment:
@twaik does the concern raised in [151a3eb](https://github.com/termux/termux-packages/commit/151a3ebdb6e409bcaa2b5aa74ae8e23447bf9e54) still apply, or are we good to reenable auto updates on Dart?

Reply:
Packages patched in this commit do not have TERMUX_PKG_SRCURL and auto-updater does not know how to handle that.

Reply:
Where does it get the Dart source from anyway?
Just had a look at the `termux_get_source()` function override in this file.

Looks like some weird Google specific thing?

Reply:
Main repo: https://github.com/dart-lang/sdk
And dependencies (kind of .gitmodules): https://github.com/dart-lang/sdk/blob/main/DEPS

Reply:
Oh well, as long as it builds.